etc/kayobe/environments/ci-multinode/inventory/group_vars/all/stackhpc-lvm.yml

```diff
@@ -32,7 +32,8 @@ stackhpc_lvm_group_rootvg_lvs_default:
   - "{{ stackhpc_lvm_lv_home }}"
```

Comment:
This file should contain the defaults, with overridden values in group_vars/controllers etc.